### PR TITLE
Build docker container with Github actions

### LINKED ISSUE #98: As developer, I want docker container to be build/published by Github Actions

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes #98

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        id: vars
+        run: |
+          # Strip git ref prefix from version
+          VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+          # Strip "v" prefix from tag name (if present at all)
+          [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
+          # Use Docker `latest` tag convention
+          [ "$VERSION" == "master" ] && VERSION=latest
+          echo ::set-output name=tag::${VERSION}
+      - name: Load image
```
Comment (annashamray) [inline .github/workflows/ci.yml:110]: Are `docker load` and `docker save` really needed here? I thought you can just do `docker build` and `docker push`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    if: github.event_name == 'push'   # Excludes PRs/forks
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download built image
+        uses: actions/download-artifact@v2
+        with:
+          name: docker-image
+
+      - name: Determine tag/commit hash
```
Comment (annashamray) [inline .github/workflows/ci.yml:100]: I think if you make `docker` and `publish` a one job, this duplicated step can be removed

**Review by annashamray — CHANGES_REQUESTED** [review]
Looks good, I have just a couple of comments.
And Could you please add badges to README (ci status and black)?

**Comment by SilviaAmAm** [comment]
I made the changes to the `ci.yml`! I will add the CI badges as part of this other ticket: https://github.com/maykinmedia/record-management-app/issues/100 :slightly_smiling_face:

**Review by annashamray — APPROVED** [review]

**Pull request merged**